docs/feature_bsp_del.md

```diff
@@ -0,0 +1,23 @@
+# Backspace Delete
+
+## Usage
+
+`KC_BSDL` will function as normal `KC_BSPC` when pressed. When either `SHIFT` key is held, it will function as `KC_DEL` instead. When both `KC_RSHIFT` and `KC_LSHIFT` are held, the output will be `SHIFT` modifier with `KC_DEL` pressed.
```

Comment:
```suggestion
`BSP_DEL` functions as a normal `KC_BSPC` when pressed. When either Shift key is held, it will function as `KC_DEL` instead. When both Left Shift and Right Shift are held, the output will be the Shift modifiers with `KC_DEL` pressed.
```